packages/devtools_app/lib/src/screens/memory/panes/control/primary_controls.dart

```diff
@@ -12,7 +12,7 @@ import '../../../../shared/utils.dart';
 import '../../memory_controller.dart';
 import '../../primitives/ui.dart';
 import '../chart/chart_pane_controller.dart';
-import 'interval_dropdown.dart';
+import 'chart_button.dart';
 
 class PrimaryControls extends StatefulWidget {
```

Comment:
let's make this widget stateless now and just pass MemoryController in as a parameter